daos/restaurant.js

```diff
@@ -0,0 +1,94 @@
+const mongoose = require("mongoose");
+// optional shortcut to the mongoose.Schema class
+const Schema = mongoose.Schema;
+
+// import related daos (to facilitate retrieval of idUser in schemaUser and idBooking in schemaBooking)
+const daoUser = require("./user");
+const daoBooking = require("./booking");
+
+/*{
+    "idRest": "658ac33fcfe93c8dbf44gb99",
+    "imageRest":
+    "name":"testsample",
+    "category": “Japanese”,
+    "location": “East”,
+    "timeOpen": "1100",
+    "timeClose": "2000",
+    "daysClose":"Thursday",
+    "address":"59 Devon Road Singapore 777888",
+    "phone":"88997788",
+    "websiteUrl":"www.testestes.com",
+    "maxPax":"30",
+    "idUser":"658ac33fcfe93c8dbf43fb28"
+    "restDest":"mid-tier jap restaurant in midtown",
+    } */
+
+const schemaRestaurant = new Schema(
+  {
+    name: {
```

Comment:
Modified some field naming such as `imageRest` `nameRest` to  `image` `name` as it is a little redundant.